gk/bpf.c

```diff
@@ -116,3 +262,32 @@ gk_load_bpf_flow_handler(struct gk_config *gk_conf, unsigned int index,
 	rte_bpf_destroy(bpf_f_init);
 	return -1;
 }
+
+int
+gk_bpf_decide_pkt(struct gk_config *gk_conf, uint8_t program_index,
+	struct flow_entry *fe, struct rte_mbuf *pkt,
+	struct gk_bpf_pkt_ctx *pctx, uint64_t *p_bpf_ret)
+{
+	struct gk_bpf_pkt_frame frame = {
+		.password = pkt_password,
+		.fe = fe,
+		.pkt = pkt,
+		.gk_conf = gk_conf,
+		.ctx = *pctx,
+	};
+	const struct gk_bpf_flow_handler *handler;
+
+
```

Comment:
Style: please remove the blank lines and define the variable directly by combining them together.